app/models/europeana/record/set.rb

```diff
@@ -25,7 +23,7 @@ class Set < ActiveRecord::Base
       validates :title, presence: true
       validate :validate_portal_urls_presence, :validate_portal_urls_format
 
-      has_settings :query_term
+      store_accessor :settings, :query_term
```

Comment:
should the key here be ':config' the same as for the other models?

Reply:
No. We will keep using `:settings` but to switch `Page` to using ActiveRecord::Store in a non-disruptive manner need to name its new serialized settings store something else first, i.e. `:config`, then later rename that back to `:settings`.